plugins/dicom_viewer/web_client/views/DicomView.js

```diff
@@ -0,0 +1,349 @@
+        restRequest({
+            path: '/item/' + this.item.get('_id') + '/dicom',
+            data: {
+                filters: 'dummy' // don't need the dicom tags, just want the sorted results
+            }
+        }).done(_.bind(function (resp) {
+            this.handleFileList(resp);
+        }, this));
+    },
+
+    handleFileList: function (files) {
+        this.files = files;
+        var slider = document.getElementById('dicom-slider');
+        slider.min = 0;
+        slider.max = this.files.length - 1;
+        slider.step = 1;
+        this.setIndex(0);
+    },
+
+    loadFile: function (file) {
+        if (file.name in this.imageDataCache) {
+            this.showCached(file);
+            return;
+        }
+        console.log(file.name);
+        if (this.xhr) {
+            this.xhr.abort();
+        }
+        const xhr = new XMLHttpRequest();
+        xhr.open('GET', apiRoot + '/file/' + file._id + '/download', true);
+        xhr.responseType = 'arraybuffer';
+        xhr.onload = _.bind(function (event) {
+            try {
+                const dataView = new DataView(xhr.response);
+                const image = daikon.Series.parseImage(dataView);
+                const imageData = createImageData(image);
+                this.imageDataCache[file.name] = imageData;
+                this.tagCache[file.name] = this.getTags(image);
+                this.showCached(file);
+            } catch (e) {
+                console.log(e);
+            }
+        }, this);
+        xhr.send();
+        this.xhr = xhr;
+    },
+
+    getTags: function (image) {
+        const result = [];
+        const tags = image.tags;
+        const keys = Object.keys(tags).sort();
+        for (let key of keys) {
+            const tag = tags[key];
+            const name = daikon.Dictionary.getDescription(tag.group, tag.element);
+
+            if (name === 'PrivateData') {
+                continue;
+            }
+            if (name.startsWith('Group') && name.endsWith('Length')) {
+                continue;
+            }
+
+            if (tag.sublist) {
+                continue;
+            } else if (tag.vr === 'SQ') {
+                continue;
+            } else if (tag.isPixelData()) {
+                continue;
+            } else if (!tag.value) {
+                continue;
+            }
+
+            const value = tag.value;
+
+            if (value.toString() === '[object DataView]') {
+                continue;
+            }
+
+            result.push(_.extend({}, {key: key, name: name, value: value}));
+        }
+        result.sort(function (a, b) {
+            a = a.name.toLowerCase();
+            b = b.name.toLowerCase();
+            return a.localeCompare(b);
+        });
+        return result;
+    },
+
+    showCached: function (file) {
+        document.getElementById('dicom-filename').innerHTML = file.name;
+        document.getElementById('g-dicom-tags').innerHTML = TagsTemplate({
```

Comment:
We should generally use `this.$` inside our Backbone views, which limits the scope of the selector to only the subtree of the widget's container (`this.$el`).